Sources/Models/Controllers/Frameworks/AVPlayerViewController.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  AVPlayerViewController.swift
+//  IBDecodable
+//
+//  Created by phimage on 04/04/2018.
+//
+
+import SWXMLHash
+
+public struct AVPlayerViewController: XMLDecodable, ViewControllerProtocol {
+
+    public let elementClass: String = "AVPlayerViewController"
+    public let id: String
+    public let customClass: String?
+    public let customModule: String?
+    public let customModuleProvider: String?
+    public var storyboardIdentifier: String?
+    public let layoutGuides: [ViewControllerLayoutGuide]?
+    public let userDefinedRuntimeAttributes: [UserDefinedRuntimeAttribute]?
+    public let view: View?
+    public var rootView: ViewProtocol? { return view }
```

Comment:
I think we can delete the `rootView` property?

Reply:
I did not map rootView like I do manually in NavigationController

For table we have
```swift
    public let tableView: TableView?
    public var rootView: ViewProtocol? { return tableView }
```

The current code will not decode well the root view
For CollectionView we must decode a `collectionView`

I think another PR could fix that

Reply:
Thanks for all your help on this. Made big progress on the SVG project. Will open source in a few days 😂